lib/ClientBase.js

```diff
@@ -240,6 +240,9 @@ ClientBase.prototype._getAllHttp = function(opts, callback, headers) {
     if (this.hasField(opts, 'order')) {
       args.order = opts.order;
     }
+    if (path === 'transactions') {
```

Comment:
🎨 Might be worth leaving a comment here about why we added this, just so we know this is CTC added code :)